client/components/seo-preview-pane/index.jsx

```diff
@@ -62,6 +64,29 @@ const getPostImage = ( post ) => {
 		: null;
 };
 
+const getSeoExcerptForPost = ( post ) => {
+	if ( ! post ) {
+		return null;
+	}
+
+	return formatExcerpt( find( [
+		PostMetadata.metaDescription( post ),
+		post.excerpt,
+		post.content
+	], identity ) );
+};
+
+const getSeoExcerptForSite = ( site ) => {
+	if ( ! site ) {
+		return null;
+	}
+
+	return formatExcerpt( find( [
+		site.options.seo_meta_description,
```

Comment:
This looks fragile. How about using a safe getter like `lodash.get()`?

You could also use something like `at()` since we're only pulling out properties of a single object…

```js
import { at } from 'lodash';

return formatExcerpt( find( at( site, 'options.seo_meta_description', 'description' ), identity ) );
```